server/mediaserver.go

```diff
@@ -312,6 +304,49 @@ func authenticateStream(url string) (*authWebhookResponse, error) {
 	return &authResp, nil
 }
 
+func jsonProfileToVideoProfile(resp *authWebhookResponse) ([]ffmpeg.VideoProfile, error) {
+	profiles := []ffmpeg.VideoProfile{}
+	for _, profile := range resp.Profiles {
+		name := profile.Name
+		if name == "" {
+			name = "webhook_" + common.DefaultProfileName(
+				profile.Width,
+				profile.Height,
+				profile.Bitrate)
+		}
+		var gop time.Duration
+		if profile.GOP != "" {
+			if profile.GOP == "intra" {
+				gop = ffmpeg.GOPIntraOnly
+			} else {
+				gopFloat, err := strconv.ParseFloat(profile.GOP, 64)
+				if err != nil || gopFloat <= 0.0 {
```

Comment:
Looks like we can avoid the nested error check here with something like:

```
if err != nil {
    return nil, err
}

if gopFloat <= 0.0 {
    return nil, errors.New("invalid gop value")
}
```

Reply:
Fixed in 4b17b6ca9b4e86f1e4c5d5648be235e0580fafc6